components/browser/menu/src/main/java/mozilla/components/browser/menu/item/WebExtensionBrowserNestedMenuItem.kt

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.menu.item
+
+import android.content.Context
+import android.view.View
+import android.widget.TextView
+import androidx.annotation.ColorRes
+import androidx.annotation.DrawableRes
+import androidx.appcompat.widget.AppCompatImageView
+import androidx.core.content.ContextCompat
+import mozilla.components.browser.menu.BrowserMenu
+import mozilla.components.browser.menu.R
+import mozilla.components.concept.menu.candidate.ContainerStyle
+import mozilla.components.concept.menu.candidate.DrawableMenuIcon
+import mozilla.components.concept.menu.candidate.TextMenuCandidate
+import mozilla.components.concept.menu.candidate.TextStyle
+
+/**
+ * A menu item for displaying text and an image icon with nested sub menu capabilities.
+ *
+ * @param label The visible label of this menu item.
+ * @param imageResource ID of a drawable resource to be shown as icon.
+ * @param iconTintColorResource Optional ID of color resource to tint the icon.
+ * @param textColorResource Optional ID of color resource to tint the text.
+ */
+@Suppress("LongParameterList")
+class WebExtensionBrowserNestedMenuItem(
```

Comment:
nit: Unlike `WebExtensionsBrowserMenuItem`, there's actually nothing specific to web extensions in this nested menu item implementation? It doesn't reference anything related to extensions. Should it just be called `BrowserNestedMenuItem` or `DefaultBrowserNestedMenuItem` if you want to keep the name of the abstract class?